_data/contributors.yml

```diff
@@ -80,4 +80,17 @@ MaheshtheDev:
       github: "MaheshtheDev"
       twitter: "MaheshtheDev"
       instagram: "MaheshtheDev" 
+      
+usman:
+  name: "Usman Sabuwala" 
+  role: "Content creator & Full stack dev"
+  avatar: "https://lh5.googleusercontent.com/nTLyy54pY21okKiOfoyx84ZLRI6WiXYR5qHlBOS36DUT-qoZ0qCuiwoQjhBlsm4ngjEhdoKk0NlpqfBPYe1u=w1872-h961"
```

Comment:
Hey! This profile image link doesn't work. Could you please use https://imgbb.com to host a custom image? We use GitHub images because that gets easier for people to change their profile pic. But any custom hosting is fine :)